modules/nextflow/src/main/groovy/nextflow/k8s/K8sDriverLauncher.groovy

```diff
@@ -49,6 +50,7 @@ import org.codehaus.groovy.runtime.MethodClosure
 @Slf4j
 class K8sDriverLauncher {
 
+    private String deploymentName = "Pod"
```

Comment:
`deploymentName` is confusing because deployment is another type of k8s resource. Perhaps `resourceType` instead? Also throughout the rest of the code

Reply:
ok, changed